Documentation/clustering.md

```diff
@@ -300,6 +300,8 @@ infra2.example.com.	300	IN	A	10.0.1.12
 
 etcd cluster members can listen on domain names or IP address, the bootstrap process will resolve DNS A records.
 
+Please make sure the resolved address in `-initial-advertise-peer-urls` matches the one of the resolved addresses in the SRV targets. That is how an etcd server finds out if it belongs to the cluster defined in the SRV records.
```

Comment:
"**The** etcd server **reads the resolved address to find** out if it belongs to the cluster defined in the SRV records."